client/starwhale/core/dataset/cli.py

```diff
@@ -101,7 +101,7 @@ def _build(
     print(config.handler)
 
     config.do_validate()
-    view.build(workdir, config)
+    return view.build(workdir, config)
```

Comment:
what's the meaning of terminal client return?

the terminal client only prints some content to stdout/stderr. If the test program should process the output, grep the output or output `-o json` maybe a better choice.

Reply:
#### why not -o json is used
Before all, `-o json` maybe a better choice indeed, which was considered to be implemented at first. But not the same as `info` or `list`, the bundle building process is deeply copuled with `progres bar`, there is a lot of work to do to implement the `-o json ` feature. I am afraid i can't make it done before release 0.3.3. 

On another hand, `-o json` means no log, which is hard for us to debug.

#### what's the meaning of terminal client return?
This could be removed now.

In order to improve the code coverage, I directlly called the `click` function in the e2e script, so the result is returned. But I forget to remove it when I turn to call the api in `view`.